src/styles/overrides/_calendar.scss

```diff
@@ -0,0 +1,132 @@
+.SECalendarKeyword {
+  .SEApplicationSubmit {
+    @extend .dg_button;
+  }
+  .SEApplicationText {
+    margin-bottom: $default-margin;
+  }
+}
+
+.SECalendarSmallCal {
+  overflow-x: auto;
+
+  table {
+    border: $border-width-small solid $gray-light;
+    color: $gray-dark;
+    font-family: $default-heading-font;
+    font-size: $smallest-heading-font-size;
+    margin-bottom: $default-margin;
+    text-align: center;
+    width: 100%;
+    th {
+      color: $black;
+      font-family: $default-heading-font;
+      font-size: $small-heading-font-size;
+      font-weight: $font-medium;
+    }
+    thead {
+      th {
+        background-color: $white;
+        border-bottom: $border-width-large solid $blue;
+        border-left: none;
+        padding: $padding-small;
+
+        &#SECalendarSmallCalMonth {
+          a {
+            text-decoration: none;
+          }
+        }
+
+        &#SECalendarSmallCalPrevious {
+          a {
+            font-size: 0;
+            text-decoration: none;
+            &:before {
+              position: relative;
+              font-size: $breadcrumb-icon-size;
+              font-weight: $font-bold;
+              content: $icon-caret-left;
+              font-family: $font-awesome-pro;
+              vertical-align: baseline;
+              left: 10px;
+            }
+          }
+        }
+
+        &#SECalendarSmallCalNext {
+          a {
+            font-size: 0;
+            text-decoration: none;
+            &:after {
+              position: relative;
+              font-size: $breadcrumb-icon-size;
+              font-weight: $font-bold;
+              content: $icon-caret-right;
+              font-family: $font-awesome-pro;
+              vertical-align: baseline;
+              right: 10px;
+            }
```

Comment:
shouldnt this be $padding-small or $margin-small

Reply:
@tmccoy529 yea i think using spacing variable makes sense here.

Reply:
@tmccoy529 Looks like this has been resolved if it is referring to line 67.

Reply:
@theilman7 Yeah I think it was fixed. Im not sure but I think the only things left were trying to consolidate code since it looks like we might have some repeat styles.